telenordigital/ProtoMan
# Gather user data

implemented a simple activity with an animation that is fired while gathering data about the user

https://jira.skunk-works.no/browse/IDBIO-36

How to test:
Fire up the app on your favorite real device (emulators do not support bluetooth)
Allow the application to use your camera etc (this should already be done in another activity when enrolling perhaps?)
Restart the application and see that you are now presented with a beautiful rotating star and see that the app does not crash. You should see information about the data gathering in your console if you are debugging
